tests/bpftrace.cpp

```diff
@@ -376,20 +391,39 @@ TEST(bpftrace, add_probes_uprobe_wildcard_for_target)
       .Times(1);
 
   ASSERT_EQ(0, bpftrace->add_probe(*probe));
-  ASSERT_EQ(4U, bpftrace->get_probes().size());
   ASSERT_EQ(0U, bpftrace->get_special_probes().size());
 
   std::string probe_orig_name = "uprobe:*:*open";
-  check_uprobe(
-      bpftrace->get_probes().at(0), "/bin/bash", "first_open", probe_orig_name);
-  check_uprobe(
-      bpftrace->get_probes().at(1), "/bin/sh", "first_open", probe_orig_name);
-  check_uprobe(
-      bpftrace->get_probes().at(2), "/bin/sh", "second_open", probe_orig_name);
-  check_uprobe(bpftrace->get_probes().at(3),
-               "/proc/1234/exe",
-               "third_open",
-               probe_orig_name);
+  if (bpftrace->has_uprobe_multi())
+  {
+    ASSERT_EQ(3U, bpftrace->get_probes().size());
+    check_uprobe(
+        bpftrace->get_probes().at(2), "/bin/bash", "*open", probe_orig_name);
```

Comment:
nit: why the reverse index checking order? 2, 1, 0

Reply:
ah, my mistake, thanks